README.md

```diff
@@ -258,35 +345,48 @@ An example of Range is:
+http://localhost:7000/blog/?filter_range_date_gte=2014-01
+and you use `search_with_url_arguments()`,
+you'll get a queryset of objects where the 'date' field is in January, 2014, or later.
 
-URL query string filters are convenient for many of the filtered queries you'll need to run, but often there are cases where you'll need more flexibility.
+URL query string filters are convenient for many of the filtered queries you'll need to run,
+but often there are cases where you'll need more flexibility.
 
 #### More complex filters
 
-By default, `search_with_url_arguments()` uses the default query parameters defined in the _queries/object-name.json file, then mixes them in with any additional arguments from the URL query string in addition to what is passed into the function itself.
+By default, `search_with_url_arguments()` uses the default query parameters
+defined in the _queries/object-name.json file,
+then mixes them in with any additional arguments
+from the URL query string in addition to what is passed into the function itself.
 
-The list of available arguments are outlined in elasticsearch-py's [search method](http://elasticsearch-py.readthedocs.org/en/master/api.html#elasticsearch.Elasticsearch.search).
+The list of available arguments are outlined in elasticsearch-py's
+[search method](http://elasticsearch-py.readthedocs.org/en/master/api.html#elasticsearch.Elasticsearch.search).
 
-The most common ones we use are size (to change the number of results returned) and q (to query based on specific fields).
+The most common ones we use are size (to change the number of results returned)
+and q (to query based on specific fields).
 
 When using q, you'll need to use the [Lucene Query Parser Syntax](http://lucene.apache.org/core/2_9_4/queryparsersyntax.html).
 
-Here is an example of using q: 
+Here is an example of using q:
 
 ```
 {% set events_jan2014 = queries.calendar_event.search_with_url_arguments(q="dtstart:[2014-01-01 TO 2014-01-31]") %}
 ```
 
-This will return a queryset of calendar_event objects which, for the field 'dtstart', have a date in January, 2014.
+This will return a queryset of calendar_event objects which,
+for the field 'dtstart', have a date in January, 2014.
 
-## How this repo is versioned
+----
 
-We use an adaptation of [Semantic Versioning 2.0.0](http://semver.org).
-Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
+## Open source licensing info
+1. [TERMS](TERMS.md)
+2. [LICENSE](LICENSE)
+3. [CFPB Source Code Policy](https://github.com/cfpb/source-code-policy/)
 
-- The MAJOR number will be incremented for major redesigns that require the user
-  to relearn how to accomplish tasks on the site.
-- The MINOR number will be incremented when new content or features are added.
-- The PATCH number will be incremented for all other changes that do not rise
-  to the level of a MAJOR or MINOR update.
+
+----
+
+## Credits and references
+
```

Comment:
I believe https://github.com/cfpb/cfgov-refresh/commit/68a6dc2ea97cf08a9988d52cb6df49d77149feaf fixes this?